core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1809,7 +1811,7 @@ private[spark] class BlockManager(
     blocksToRemove.size
   }
 
-  def decommissionBlockManager(): Unit = synchronized {
+  private[spark] def decommissionSelf(): Unit = synchronized {
```

Comment:
Didn't follow the need for the name change.

Reply:
It conflicts with the newly added "decommissionBlockManager()".